CSS/CSS.sublime-syntax

```diff
@@ -125,7 +121,31 @@ contexts:
           scope: punctuation.definition.comment.css
           pop: true
 
-  custom-media:
+  at-charset:
+    - match: \s*((@)charset\b)\s*
+      captures:
+        1: keyword.control.at-rule.charset.css
+        2: punctuation.definition.keyword.css
+      push:
+        - meta_scope: meta.at-rule.charset.css
+        - include: at-rule-punctuation
+        - include: literal-string
+
+  # @counter-style
+  # https://drafts.csswg.org/css-counter-styles-3/#the-counter-style-rule
+  at-counter-style:
+    - match: \s*((@)counter-style\b)\s+(?:(?i:\b(decimal|none)\b)|({{ident}}))\s*(?={)
+      captures:
+        1: keyword.control.at-rule.counter-style.css
+        2: punctuation.definition.keyword.css
+        3: invalid.illegal.counter-style-name.css
+        4: entity.other.counter-style-name.css
```

Comment:
Would this scope `entity.other.counter-style-name.css`, be better as `entity.name.counter-style.css`?

Any changes would also apply to [entity.other.animation-name.css](https://github.com/Briles/Packages/blob/fdf35f282921b3532db9b2359820655c70307c99/CSS/CSS.sublime-syntax#L218)